src/models/MenuSet.php

```diff
@@ -51,9 +56,9 @@ class MenuSet extends DataObject
      * @var array
      */
     private static $db = [
-        'Title' => 'Varchar(255)',
-        'Slug' => 'Varchar(255)',
-        'Nested' => 'Boolean'
+        'Title' => DBVarchar::class,
+        'Slug' => DBVarchar::class,
+        'Nested' => DBBoolean::class,
```

Comment:
I thought it was the new recommended standard, but it seems the official documentation still uses the old magic strings. I can revert this part if you like.

I prefer to use explicit class references over magic strings because it helps avoid potential issues like typos etc. through IDE hinting, and allows clicking through to class definition to view methods and the like.